commonmarker.gemspec

```diff
@@ -14,8 +14,7 @@ Gem::Specification.new do |spec|
   spec.homepage = "https://github.com/gjtorikian/commonmarker"
 
   spec.required_ruby_version = "~> 3.1"
-  # https://github.com/rubygems/rubygems/pull/5852#issuecomment-1231118509
-  spec.required_rubygems_version = ">= 3.3.22"
+  spec.required_rubygems_version = "~> 3.4"
```

Comment:
Did you intend to drop support for all rubies < 3.4 in version 2.0.2? That's what I'm seeing, anyway.

Reply:
Nevermind, I misread this.